WirelessModules/src/wireless_module.py

```diff
@@ -0,0 +1,13 @@
+class WirelessModule:
+    def __init__(self):
+        super().__init__()
+        self.sensors = []
+
+    def add(self, sensor):
+        self.sensors.append(sensor)
+    
+    def read_sensors(self):
+        readings = []
+        for sensor in self.sensors:
+            readings += sensor.read()
```

Comment:
Had no idea you could append elements to an array with tuples like that. Super cool!